.travis.yml

```diff
@@ -11,3 +11,5 @@ script:
   - flake8
   - find . -name "*.py" | xargs mypy
   - pytest -vv --cov=barrage
+after_success:
```

Comment:
Need to pip install `coveralls` or add it to your `extras_require` for test in setup.py

Reply:
Beat me by a couple of minutes.

Let's hope that does the trick :)